.travis.yml

```diff
@@ -9,6 +9,7 @@ branches:
   except:
     - piptools-ignore-patch
 install:
+  - "python -m pip install pip --upgrade"
```

Comment:
The one i added on .travis? None, anyway i will correct the package itself first and we will see about travis and tests ;)